_pytest/skipping.py

```diff
@@ -178,6 +184,18 @@ def pytest_runtest_setup(item):
 
 def pytest_pyfunc_call(pyfuncitem):
```

Comment:
Just realized this is the wrong hook for this. :frowning: 

It is not executing the test at all when `strict` is True... I will fix this later tonight. :disappointed: